kore/src/Kore/Step/Simplification/Rule.hs

```diff
@@ -85,6 +85,10 @@ simplifyRulePattern rule = do
     case OrPattern.toPatterns simplifiedLeft of
         [ Conditional { term, predicate, substitution } ]
           | PredicateTrue <- predicate -> do
+            -- TODO (virgil): Dropping the substitution for equations
```

Comment:
Could you open an issue for this, if there isn't one already?

Reply:
https://github.com/kframework/kore/issues/1369